detection_rules/rule.py

```diff
@@ -53,7 +53,7 @@ class RuleMeta(MarshmallowDataclassMixin):
 
     # Optional fields
     comments: Optional[str]
-    integration: Optional[str]
+    integration: Optional[List[str]]
```

Comment:
This is a metadata change so it will only affect the rule's metadata. This is not used when creating a hash of the rule content itself. All rules, if an integration tag exists, will need to be a list.